src/utils/common.ts

```diff
@@ -61,7 +60,7 @@ export class CommonUtil {
     }
 
     public static clone(o: any): object {
-        return cloneDeep(o);
+        return { ...o };
```

Comment:
Are we sure this is correct though? Doesn't this mean that anywhere the acl library needs a deep clone it'll break? Are we sure there are no cases of this?

Reply:
Not sure yet. According to the [reason it was introduced](https://github.com/tensult/role-acl/pull/39/files), it was because someone was trying to use a "custom function to control permissions" and they were getting scope permissions. I don't believe we're doing this anywhere, but wanted to throw this up in parallel with investigating what exactly that feature is

Reply:
My guess is that on each place where we're doing eg `resource: 'Field', action: 'EDIT'` there's probably an additional parameter option that allows overriding the `AccessControl` entity which by default has these methods:

* isAllowed
* getAllowedAttributes
* allowedResources
* allowedActions
* allowedCategories

Will search our codebase if we're doing that anywhere

Reply:
Turning into a non-trivial operation to upgrade 3 y/o dependencies to get it working at all:

1. package-lock untouched in role-acl results in a checksum failure when npm installing this commit from sora/server
2. package-lock recreated in role-acl and then pushed fixes npm install from sora/server, but server crashes when first request comes in with error from an anonymous function with a 20 line stacktrace through datadog libraries

Doing neither of the above but modifying the code directly in node_modules for this library has no side effects I've been able to tell. So I'd love to figure out how to get these new deps working but that seems like it will be a hassle. Mulling this over...

Reply:
Never mind, I was doing this off the wrong branch. I opened a new PR here: https://github.com/soradotco/role-acl/pull/13, closing this one